.github/workflows/test.yml

```diff
@@ -66,6 +92,21 @@ jobs:
           docker save ovn-daemonset-f:dev > _output/image-master.tar
         popd
 
+    - name: Cache master image
+      if: steps.is_master_image_build_needed.outputs.MASTER_IMAGE_RESTORED != 'true' && success()
+      continue-on-error: true
+      run: |
+        if [ -f /tmp/image_cache/image-master.tar ]; then
```

Comment:
instead of using static paths, I would suggest to use variables. Would be easier for future changes and code maintenance.

```
env
   DIR_IMAGE_CACHE=/tmp/image_cache/
   FILE_IMAGE_CACHE=$DIR_IMAGE_CACHE/image-master.tar
```

Reply:
agreed, a small typo for these paths in some place could go unnoticed. Probably better to have defined variables.

Reply:
Alright, these points should hopefully be addressed now. Thanks for the review to both of you!